router.go

```diff
@@ -380,7 +384,13 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		// Handle OPTIONS requests
 		if r.HandleOPTIONS {
 			if allow := r.allowed(path, req.Method); len(allow) > 0 {
+				if r.OPTIONSPostHandler != nil {
+					r.OPTIONSPostHandler.ServeHTTP(w, req)
```

Comment:
Sorry. This shouldn't be there twice. I will fix this. It should only be called after. Not sure how that happened.